src/DnsRootStack.ts

```diff
@@ -57,10 +125,141 @@ export class DnsRootStack extends cdk.Stack {
       recordName: 'auth-prod',
       values: ['60066 13 2 0E517A7669408AFC5345B167EBEBC3BB0D355E48FF160EBCF46EF27189C49949'],
     });
+    new Route53.DsRecord(this, 'accp-ds-record', {
+      zone: this.cspNijmegenZone,
+      recordName: 'accp',
+      values: ['52561 13 2 07CCC7D1A28A7BF1F221F50589AF606FD807B1793F28B0D76AF7F56D32FDE0FE'],
+    });
   }
 
   temporarelyAddExistingCspNijmegenRecords() {
-    // TODO: add records
+
+    // Good to know but also present in accp.csp-nijmegen.nl hosted zone
+    // accp.csp-nijmegen.nl	TXT	Simple	-	"v=spf1 include:amazonses.com ~all"
+    // accp.csp-nijmegen.nl	MX	Simple	- 10 feedback-smtp.eu-west-1.amazonses.com
+
+    // Not relevant i think (literal slashes in there)
+    // mail.csp-nijmegen.nl	TXT	Simple	-	"\"v=spf1 include:amazonses.com ~all\""
+
+    // NOTE: A and AAAA records must be configured manually!
+
+    // TEMP records mijn nijmegen
+    new Route53.DsRecord(this, 'mijn-nijmegen-ds', {
+      zone: this.cspNijmegenZone,
+      recordName: 'mijn',
+      values: ['60066 13 2 932CD585B029E674E17C4C33DFE7DE2C84353ACD8C109760FD17A6CDBD0CF3FA'],
+    });
+    new Route53.NsRecord(this, 'mijn-nijmegen-ns', {
+      zone: this.cspNijmegenZone,
+      recordName: 'mijn',
+      values: [
+        'ns-674.awsdns-20.net',
+        'ns-1880.awsdns-43.co.uk',
+        'ns-1091.awsdns-08.org',
+        'ns-160.awsdns-20.com',
+      ],
+    });
+
+    // Other records
+    new Route53.TxtRecord(this, 'temp-cdn-txt', { // I have no clue what this does
```

Comment:
ik vermoed dat deze voor domein validatie gebruikt is/wordt

Reply:
Daar staat inderdaad een comment met "validation record for cnd.csp-nijmegen.nl (for prod environment)"